pysb/bng.py

```diff
@@ -103,12 +104,16 @@ class BngBaseInterface(object):
     @abc.abstractmethod
     def __init__(self, model, verbose=False, cleanup=False,
                  output_prefix=None, output_dir=None):
+        self._base_file_stem = 'pysb'
         self.verbose = verbose
         self.cleanup = cleanup
         self.output_prefix = 'tmpBNG' if output_prefix is None else \
             output_prefix
-        self.generator = BngGenerator(model)
-        self._check_model()
+        if model:
```

Comment:
Is there a use case for calling the constructor with model=None? A lot of the code in this class can be simplified if model is guaranteed to be defined.